### PR TITLE
Convert nav markup to .erb

### PR DESCRIPTION
**Pull request by pixelflips**
- [x] Create a markup file for nav.
- [x] Update the preview and markup files.

### CODE REVIEW

File: app/views/sage/examples/objects/nav/_markup.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="sage-nav">
```
Comment (teenwolfblitzer) [inline app/views/sage/examples/objects/nav/_markup.html.erb]: Hmm, not sure why this wasn't the case before, but this `div` should be a `nav` element with an `aria-label` indicating the region (see the actual sidebar for an example). Those two items will help distinguish the landmark in screen readers.

Ideally, we'd also use an unordered list grouping the nav items together, but that's better addressed in a follow-up.

File: app/views/sage/examples/objects/nav/_markup.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  <% nav_links.each do |link| %>
+  <a href="#" class="sage-nav__link <%= link[:is_sublink] ? "sage-nav__link--sub" : "" %>"><%= link[:text] %></a>
```
Comment (teenwolfblitzer) [inline app/views/sage/examples/objects/nav/_markup.html.erb]: Should we also make the link URLs here editable in the options?

**Review by teenwolfblitzer — COMMENTED** [review]
One item to address and one to consider, nice work so far!

File: app/views/sage/examples/objects/nav/_markup.html.erb (hunk 1 above)
Comment (pixelflips) [reply app/views/sage/examples/objects/nav/_markup.html.erb]: @monkeypox8 thanks for the info. I'll update the element to nav and add the `aria-label` and then I'll create an issue in the backlog for the `ol` update.

File: app/views/sage/examples/objects/nav/_markup.html.erb (hunk 2 above)
Comment (pixelflips) [reply app/views/sage/examples/objects/nav/_markup.html.erb]: Sure. Will adjust.

**Review by teenwolfblitzer — APPROVED** [review]
👍 🔥 Nice work!

**Pull request merged**